nlopt/config.h

```diff
@@ -55,12 +55,18 @@
 /* Define to 1 if you have the <sys/stat.h> header file. */
 #define HAVE_SYS_STAT_H 1
 
+/* Define to 1 if you have the <sys/time.h> header file. */
+#define HAVE_SYS_TIME_H 1
```

Comment:
Windows does not have sys/time, so this always fails.  We need to be able to maintain one file that when checked out, compiles on both Windows, Linux, and OSX.  I will add a conditional statement around this that sets correctly for windows.